docs/Datadog.Trace/README.md

```diff
@@ -0,0 +1,220 @@
+
+## Getting Started
+
+1. Configure the Datadog agent for APM [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core#configure-the-datadog-agent-for-apm).
+2. For automatic instrumentation, install and enable the tracer [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core/?tab=windows#install-the-tracer). 
+3. Configure custom instrumentation, as shown below
+4. [View your live data on Datadog](https://app.datadoghq.com/apm/traces).
+
+### Configuring Datadog in code
+
+There are multiple ways to configure your application, for example using Environment variables, _web.config_, or _datadog.json_, [as described in our documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/dotnet-core/#configuration). This NuGet package also allows you to configure settings in code.
+
+To override configuration settings, create an instance of `TracerSettings`, and pass it to the static `Tracer.Configure()` method:
+
+```csharp
+using Datadog.Trace;
+
+// Create a settings object using the existing 
+// environment variables and config sources
+var settings = TracerSettings.FromDefaultSources();
+
+// Override a value
+settings.GlobalTags.Add("SomeKey", "SomeValue");
+
+// Replace the tracer configuration
+Tracer.Configure(settings);
+```
+
+Calling `Tracer.Configure()` will replace the settings for all subsequent traces, both for custom instrumentation and for automatic instrumentation.
+
+> :warning: Replacing the configuration should be done once, as early as possible in your application. 
+
+ ### Create custom traces
+
+To create and activate a custom span, use `Tracer.Instance.StartActive()`. If a trace is already active (when created by automatic instrumentation, for example), the span will be part of the current trace. If there is no current trace, a new one will be started.
+
+> :warning: Ensure you dispose of the scope returned from StartActive. Disposing the scope will close the span, and ensure the trace is flushed to Datadog once all its spans are closed. 
+
+```csharp
+using Datadog.Trace;
+
+// Start a new span
+using (var scope = Tracer.Instance.StartActive("custom-operation")
+{
+    // Do something
+}
+```
+
+## Release Notes
+
+You can view the [notes for the latest release on GitHub](https://github.com/DataDog/dd-trace-dotnet/releases).  
+
+## Upgrading from 1.x to 2.0
+
+Version 2.0 of this package introduced a number of breaking changes to the API which allowed various performance improvements, added new features, and deprecated problematic ways of using the package. Most of these changes will not require any changes to your code, but some patterns are no longer supported or recommended.
+
+This section describes some of the most important breaking changes. For full details see [the release notes on GitHub](https://github.com/DataDog/dd-trace-dotnet/releases/tag/v2.0.0).
+
+## Updates for supported .NET versions
```

Comment:
I think we should move this section up so that all the `###` subsections below fall under `## Upgrading from 1.x to 2.0`

Reply:
I actually meant for this to be a `###` under the `Upgrading from 1.x to 2.0` header, as it describes a breaking change. Do you think it's better as its own section entirely?

Reply:
Looking at this comment 23 days later and I can't remember why I suggested this 😅

Reply:
> I actually meant for this to be a `###`

But it's a `##` right now. Do we need to fix it?

Reply:
Fixed in eec9224d1c952447029cbcb1326a8128b2e3fcac